src/harness/fourslashImpl.ts

```diff
@@ -334,6 +342,14 @@ namespace FourSlash {
                 if (!compilationOptions.noLib) {
                     this.languageServiceAdapterHost.addScript(Harness.Compiler.defaultLibFileName,

```

Comment:
The below code is identical to the front, is there any possibilities to reuse it?